net/bmx7/files/etc/init.d/bmx7

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh /etc/rc.common
+START=91
+USE_PROCD=1
+
+BIN=/usr/sbin/bmx7
+CONF=/etc/config/bmx7
+
+start_service() {
+	cd /root/
```

Comment:
Shellcheck reports this error: https://github.com/koalaman/shellcheck/wiki/SC2164

Just add || exit

Reply:
Thanks, updated